content/source/docs/cloud/users-teams-organizations/single-sign-on/okta.html.md

```diff
@@ -48,3 +50,25 @@ Be sure to copy the metadata URL (from the final step of configuring Okta) befor
 6. Your Okta SSO configuration is complete and ready to [use](../single-sign-on.html#using-sso).
 
     ![sso-settings](../images/sso/settings-okta.png)
+
+## Configuration - Team Mapping (Okta)
+1. Complete all steps outlined in the [Configuration (Okta)](./okta.html#configuration-okta-) section above, and take note of the default team mapping behavior as described [here](../single-sign-on.html#managing-team-membership-through-sso).
+
+2. Edit your Terraform Cloud Okta Application and complete the following steps:
+* Expand the `Attributes` section of the Application configuration (under the `Sign On` tab):
+
+    ![sso-okta-attribute-settings](../images/sso/okta-attribute-settings.png)
+
+* Set the `Group Attribute` Statements to the following:
+    * Name: `MemberOf`
+    * Name format: `Basic`
+    * Filter: `Matches regex`
+    * Filter value: `.*`
+
+    ![sso-okta-team-mapping](../images/sso/okta-team-mapping.png)
+
+Once these configure steps have been completed, **all** Okta groups to which a given User belongs will be passed in the SAML assertion upon login to Terraform Cloud, which means that User will get added automatically to any Teams within Terraform Cloud for which there’s an **exact** name match.  
```

Comment:
Normal nouns like "user" and "team" should not be capitalized, because they are not brand names or other proper nouns.

Reply:
Updated.